css/promo.css

```diff
@@ -0,0 +1,151 @@
+.promo__bg {
+    padding-top: 118px;
+
+    background: url(../assets/img/header_cake_sm.png) no-repeat center center;
+    background-position-y: -50px;
+}
+
+.promo__cons-text {
+    font-family: "Circe Regular";
+}
+
+.promo {
+    background: #fbebea;
+}
+
+.promo__title {
+    width: 180px;
+    margin: 0 auto;
+    margin-bottom: 20px;
+
+    text-align: center;
+
+    font-family: "Playfair Display", serif;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 35px;
+}
+
+.promo__text {
+    margin: 0;
+    padding-bottom: 20px;
+
+    text-align: center;
+
+    font-family: "Intro Script R H2 Base";
+    font-size: 22px;
+}
+
+.promo__cons-text {
+    width: 200px;
+    margin-left: 10px;
+
+    font-size: 14px;
+}
+
+.promo__item {
+    margin-bottom: 13px;
+}
+
+.promo__item--present {
+    margin-bottom: 40px;
+}
+
+.promo__cons-img {
+    display: none;
+}
+
+@media (min-width: 576px) {
+    .promo__cons {
+        width: 250px;
+    }
+}
+
+@media (min-width: 770px) {
+    .promo__title {
+        width: 300px;
+        margin: 0;
+        padding-left: 50px;
+
+        text-align: left;
+
+        font-size: 60px;
+        line-height: 55px;
+    }
+
+    .promo {
+        position: relative;
+
+        height: 675px;
+
+        background: url(../assets/img/promo-cake-md.png) no-repeat;
+        background-position-x: 100%;
+        background-position-y: 15px;
+    }
+
+    .promo__bg {
+        background: none;
+    }
+
+    .promo__text {
+        padding: 30px 0 0 50px;
+
+        text-align: left;
+
+        font-size: 36px;
+    }
+
+    .promo .button {
+        margin: 35px 0 0 50px;
+    }
+
+    .promo__cons--md {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        width: 460px;
+        margin: 130px 0 0 50px;
+    }
+
+    .promo__cons-text {
+        width: 155px;
+
+        font-size: 14px;
+        font-weight: 700;
+    }
+
+    .promo__item {
+        display: flex;
+        align-items: center;
+    }
+
+    .promo__bg {
+        padding-top: 100px;
+    }
+
+    .promo__img {
+        display: none;
+    }
+
+    .promo__cons-img {
+        display: block;
+    }
+
+    .promo__container-cons {
+        display: flex;
+        flex-direction: column-reverse;
+    }
+
+    .promo__cons {
+        display: flex;
+        align-items: center;
+
+        padding: 145px 0 0 30px;
+    }
+
+    .promo__item--present,
```

Comment:
No need to give modifier same styles as element already has

Reply:
done